### PR TITLE
Cufflinks 2.2.1 samtools 0.1.19 goolf 1.7.20

### PR DESCRIPTION
**Pull request by jprorama**
This is an update to Cufflinks to support the goolf-1.7.20 toolchain. It builds successfully on CentOS 7.

It bumps the tool chain version, Boost, Eigen and zlib. The SAMtools version remains the same.

Moved pull request to target develop branch

### CODE REVIEW

**Comment by rjeschmi** [comment]
Did you close this one too? Was there a problem?

**Comment by jprorama** [comment]
This one should still be open.  I just "moved"  PR2050 to PR2058 in order to submit it against the develop branch as requested.

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name = 'Cufflinks'
+version = '2.2.1'
+
+homepage = 'http://cufflinks.cbcb.umd.edu/'
```
Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb]: change to http://cole-trapnell-lab.github.io/cufflinks/ ?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/66d8726389508ee94207 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8f87beb0088ceb64dacd for a full test report.

**Comment by boegel** [comment]
lgtm, @rjeschmi?

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb (hunk 1 above)
Comment (rjeschmi) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb]: Yeah, this is the homepage that has been updated in the newer foss and intel toolchains. It should probably be updated here too.

**Comment by rjeschmi** [comment]
I don't see any problem. I'm running a test too, but should be good to go.

**Comment by rjeschmi** [comment]
Test report by @rjeschmi
**SUCCESS**
Build succeeded for 7 out of 7 (2 easyconfigs in this PR)
Linux centos 6.6, Intel(R) Xeon(R) CPU           E5440  @ 2.83GHz, Python 2.6.6
See https://gist.github.com/0bd9bcdf1a0e88fb6c54 for a full test report.

**Comment by boegel** [comment]
Thanks for the feedback/testing @rjeschmi!

Going in, congratulations on your first contribution @jprorama, keep them coming!
Welcome to the wonderful world of EasyBuild. ;-)

File: easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb]: We can do this in a follow-up PR, for all Cufflinks easyconfigs.

Comment (boegel) [inline easybuild/easyconfigs/c/Cufflinks/Cufflinks-2.2.1-goolf-1.7.20.eb]: see #2060

**Pull request merged**